.travis.yml

```diff
@@ -1,8 +1,3 @@
-language: rust
-rust:
-  - stable
-  - beta
-  - nightly
-matrix:
-  allow_failures:
-    - rust: nightly
+language: nix
+
```

Comment:
I think that we should have a install section (eg `install: ./ci-install.sh`) where ci-install.sh contains the dependency installation parts of ci.sh; this would give more refined error messages when builds fail due to broken dependencies.